benchmark/src/grpc_helpers.rs

```diff
@@ -103,7 +101,7 @@ fn wait_read_requests<
         for response_result in read_stream.wait() {
             match response_result {
                 Ok(proto_resp) => {
-                    let resp_size = f64::from(proto_resp.compute_size());
+                    let resp_size = proto_resp.encoded_len() as f64;
```

Comment:
do you know why we're using floats oO???

Reply:
Yeah i have no idea, I think it may have something to do with Prometheus representing counters as floats